UI/js/utils/utils.js

```diff
@@ -98,12 +98,13 @@ $(document).ready(function(){
       // pick a random image and assign it to the body element
       var ramdomImage = _.sample(data);
 
-      //need to get absolute path to deal with Chrome/Safari
-      document.body.style.backgroundImage = 'url(' + window.location + 'external/image/' + ramdomImage + ')';
+      document.body.style.backgroundImage = 'url(external/image/' + ramdomImage + ')';
+      document.body.style.backgroundColor = '#444444';
     })
     .fail(function() {
       // select a default image and assign it to the body element
-      document.body.style.backgroundImage = 'url(' + window.location + 'data/images/back/default.jpg' + ')';
+      document.body.style.backgroundImage = 'url(data/images/back/default.jpg)';
+      document.body.style.backgroundColor = '#444444';
```

Comment:
Good point regarding the background color - it shall be done. Regarding the adding/deleting - the new ones fade to #444 to deal with cases where the content was taller than the images.